html/pfappserver/lib/pfappserver/Form/Config/Network.pm

```diff
@@ -50,14 +50,35 @@ has_field 'dhcp_max_lease_time' =>
    required_when => { 'fake_mac_enabled' => sub { $_[0] ne '1' } },
    messages => { required => 'Please specify the maximum DHCP lease time.' },
   );
+has_field 'ip_reserved' =>
+  (
+   type => 'Text',
+   label => 'IP Addresses reserved',
+   required_when => { 'fake_mac_enabled' => sub { $_[0] ne '1' } },
+   messages => { required => "Please specify the DNS server's IP address(es)." },
+   tags => { after_element => \&help,
+             help => 'Comma delimited range or IP Addresses' },
+  );
 has_field 'dns' =>
   (
    type => 'IPAddresses',
    label => 'DNS Server',
    required_when => { 'fake_mac_enabled' => sub { $_[0] ne '1' } },
    messages => { required => "Please specify the DNS server's IP address(es)." },
    tags => { after_element => \&help,
-             help => 'Should match the IP of a registration interface or the production DNS server if the network is Inline L3' },
+             help => 'Should match the IP of a registration interface or the production DNS server(s) if the network is Inline L2/L3 (space delimited list of IP addresses)' },
+  );
+has_field 'force_dns' =>
```

Comment:
Shouldn't this field only show when running a cluster ?

Seems useless otherwise but I can be wrong

Reply:
It's for administrator who wants to use another dns server than packetfence.